src/main/kotlin/org/wycliffeassociates/resourcecontainer/DirectoryAccessor.kt

```diff
@@ -0,0 +1,25 @@
+package org.wycliffeassociates.resourcecontainer
+
+import java.io.BufferedReader
+import java.io.BufferedWriter
+import java.io.File
+
+class DirectoryAccessor(private val file: File) : IResourceContainerAccessor {
```

Comment:
I'd name the val root since that's what you're treating it as

Reply:
done